hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -437,6 +451,18 @@ private Table newHmsTable(TableMetadata metadata) {
             null,
             TableType.EXTERNAL_TABLE.toString());
 
+    // hive2 does not support setting table ownership type, therefore we leave it as
```

Comment:
I'm not really sure about this approach. If we can't add ownership type into HMS and have to use a table property for this purpose would the clients of Iceberg respect this table property or would it need extra implementation to get the ownership type from the property?

Reply:
It needs extra implementation to get the ownership type from the property.

Hopefully, let's not upgrade Iceberg's hive dependency version just because of this. I actually tried, and found that is no small task.

This is some temporary solution to compensate for the fact that hive2 does not support table ownership type. I'm open to other suggestions.

Reply:
Decided to not support group ownership for tables here since hive2 does not support it. Hopefully this will simplify this PR. So this thread may not be relevant anymore.